lmdeploy/lite/quantization/calibration.py

```diff
@@ -291,8 +294,12 @@ def _search_module_scale(block, linears2scale: list, x, kwargs={}):
         org_sd = {k: v.cpu() for k, v in block.state_dict().items()}
         for ratio in range(0, n_grid):
             ratio = ratio * 1 / n_grid
-            scales = (x_max.pow(ratio) /
-                      w_mean.pow(1 - ratio)).clamp(min=1e-4).view(-1)
+            w_mean_pow = w_mean.pow(1 - ratio)
+            if w_mean_pow.min().item() == 0:
```

Comment:
Given `w_mean >= 1e-4` and `ratio in [0, 1/20, 2/20, ..., 19/20]`, w_mean_pow > 0.00015
Therefore this branch is always False. It's unnecessary to clamp it here

Reply:
w_mean contains zeros.